src/gui/tray/Notification.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+
+// Custom qml modules are in /theme (and included by resources.qrc)
+import Style 1.0
+
+Item {
+    id: notificationItem
+    property alias text: notificationMessage.text
+    signal dismissNotification()
+
+    width: parent.width - Style.trayWindowBorderWidth*2
+    height: Style.notificationHeight
+    anchors.bottom: parent.bottom
+    anchors.horizontalCenter: parent.horizontalCente
+
+    Rectangle {
+        id: roundedBackground
+        width: parent.width
+        height: parent.height
+        anchors.bottom: parent.bottom
+        color: Style.lightMessage
+        radius: Style.trayWindowRadius
+    }
+
+    Rectangle {
```

Comment:
It took me a while to understand what this one is for. Maybe a short comment would help